packages/flutter_tools/templates/create/android.tmpl/buildSrc/src/main/groovy/FlutterPlugin.groovy

```diff
@@ -0,0 +1,229 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
If we're not going to allow it to be modified, then we shouldn't include it in the template. We should just pull it in at build time.